cmd/root.go

```diff
@@ -110,7 +110,7 @@ func validateService(service string) error {
 		}
 	} else {
 		if !validServicePathFormat.MatchString(service) {
-			return fmt.Errorf("Failed to validate service name '%s'.  Only alphanumeric, dashes, forwardslashes, fullstops and underscores are allowed for service names", service)
+			return fmt.Errorf("Failed to validate service name '%s'.  Only alphanumeric, dashes, forwardslashes, fullstops and underscores are allowed for service names. Service names must not start or end with a forwardslash", service)
```

Comment:
Nitpick: only using a single space everywhere else, and "forward slash" and "full stop" are two words each.

(I know you didn't make those typos -- just flagging them while this is open.)

```suggestion
			return fmt.Errorf("Failed to validate service name '%s'. Only alphanumeric, dashes, forward slashes, full stops and underscores are allowed for service names. Service names must not start or end with a forward slash", service)
```

Reply:
Thanks for pointing it out! Added in https://github.com/segmentio/chamber/pull/347/commits/843d594527ac20891d5f088273c9c1630edbb38a (found some other usages of `fullstops` and double spaces)